src/interface/src/app/plan/create-scenarios/constraints-panel/constraints-panel.component.spec.ts

```diff
@@ -76,37 +84,19 @@ describe('ConstraintsPanelComponent', () => {
     expect(component).toBeTruthy();
   });
 
-  // it('should emit event when next button is clicked and form is valid', async () => {
-  //   spyOn(component.formNextEvent, 'emit');
-  //   const nextButton: MatButtonHarness = (
-  //     await loader.getAllHarnesses(MatButtonHarness)
-  //   )[0];
-  //   // Set form to valid state
-  //   component.constraintsForm?.get('budgetForm.estimatedCost')?.setValue('1');
-
-  //   expect(component.constraintsForm?.valid).toBeTrue();
-  //   expect(await nextButton.isDisabled()).toBeFalse();
-
-  //   // Click next button
-  //   await nextButton.click();
-
-  //   expect(component.formNextEvent.emit).toHaveBeenCalledOnceWith();
-  // });
-
-  // it('should not emit event when next button is clicked and form is invalid', async () => {
-  //   spyOn(component.formNextEvent, 'emit');
-  //   const nextButton: MatButtonHarness = (
-  //     await loader.getAllHarnesses(MatButtonHarness)
-  //   )[0];
-
-  //   expect(component.constraintsForm?.valid).toBeFalse();
-
-  //   // Click next button
-  //   await nextButton.click();
-
-  //   expect(component.formNextEvent.emit).toHaveBeenCalledTimes(0);
-  // });
-
-
+  it('should disable maxCost input when no estimatedCost input is provided', async () => {
```

Comment:
can you add a test if 0 is inputted, since it appears that one could set it to 0 per the code.

Reply:
Do we want the users to not be able to input 0? I can set the minimum to be 1 or another value

Reply:
It probably isn't so useful for users to add a maxCost of 0; I'd rather have users blank it out if they wanted to say there was no minimum vs. interpreting 0 as no-minimum.  Maybe add a todo to enforce a positive number of blank.  (I'm really surprised that angular validators don't have a greaterthan.)

Reply:
SGTM